File: wrappers/java/src/test/java/org/hyperledger/indy/sdk/ledger/RevocRegEntryRequestTest.java
package org.hyperledger.indy.sdk.ledger;

import org.hyperledger.indy.sdk.IndyIntegrationTestWithPoolAndSingleWallet;
import org.junit.Test;

import static org.junit.Assert.assertTrue;

public class RevocRegEntryRequestTest extends IndyIntegrationTestWithPoolAndSingleWallet {

	@Test
	public void testBuildRevocRegEntryRequestWorks() throws Exception {
		String expectedResult ="\"operation\": {\n" +
						"            \"type\": \"114\",\n" +
						"            \"revocRegDefId\": \"RevocRegID\",\n" +
						"            \"revocDefType\": \"CL_ACCUM\",\n" +
						"            \"value\": {\"accum\": \"false 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\"}\n" +
						"        }";

		String value = "{\"ver\":\"1.0\"," +
				"        \"value\": {\n" +
				"            \"accum\": \"false 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\"\n" +
				"        }}";

		String request = Ledger.buildRevocRegEntryRequest(DID, "RevocRegID", "CL_ACCUM", value).get();

		assertTrue(request.replaceAll("\\s+", "").contains(expectedResult.replaceAll("\\s+", "")));
	}
}

File: wrappers/dotnet/indy-sdk-dotnet-test/LedgerTests/RevocRegEntryRequestTest.cs
using Hyperledger.Indy.LedgerApi;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System.Threading.Tasks;

namespace Hyperledger.Indy.Test.LedgerTests
{
    [TestClass]
    public class RevocRegEntryRequestTest : LedgerIntegrationTestBase
    {
        [TestMethod]
        public async Task TestBuildRevocRegEntryRequestWorks()
        {
            var expectedResult =
                "\"operation\":{" +
                        "\"type\":\"114\"," +
                        "\"revocRegDefId\":\"RevocRegID\"," +
                        "\"revocDefType\":\"CL_ACCUM\"," +
                        "\"value\":{\"accum\":\"1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000\"}" +
                        "}";

           var value = "{\"ver\":\"1.0\"," +
                "        \"value\": {\n" +
                "            \"accum\": \"1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000 1 0000000000000000000000000000000000000000000000000000000000000000\"\n" +
                "        }}";

            var request = await Ledger.BuildRevocRegEntryRequestAsync(DID, "RevocRegID", "CL_ACCUM", value);
            Assert.IsTrue(request.Replace("\\s+", "").Contains(expectedResult.Replace("\\s+", "")));
        }
    }
}
